hadoop-hdds/dev-support/checkstyle/checkstyle.xml

```diff
@@ -136,6 +136,10 @@
 
         <!-- Checks for Size Violations.                    -->
         <!-- See http://checkstyle.sf.net/config_sizes.html -->
+        <module name="LineLength">
+            <property name="max" value="120"/>
+            <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
+        </module>
```

Comment:
checkstyle was upgraded in HDDS-6363.  With the new version `LineLength` is no longer a child of `TreeWalker`, it had to be moved to:

https://github.com/apache/ozone/blob/9b16de3fc1524cc263c5a14576559bec2722e7dd/hadoop-hdds/dev-support/checkstyle/checkstyle.xml#L74-L76

so with the patch we [get](https://github.com/apache/ozone/actions/runs/7156759419/job/19486838107?pr=921#step:6:404):

```
Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.2:check (default-cli) on project ozone-main: Failed during checkstyle configuration: cannot initialize module TreeWalker - TreeWalker is not allowed as a parent of LineLength Please review 'Parent Module' section for this Check in web documentation if Check is standard.
```

(Unfortunately this config error is not reflected in CI status; will create a bug for that.)

To fix this, please add 
```
        <property name="max" value="120"/>
        <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
```

in the existing `LineLength` module def.

Reply:
@adoroszlai Thank you for your help, I've move these two properties to the `LineLength` Module, PTAL.